airbyte-integrations/connectors/source-mysql-strict-encrypt/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM airbyte/integration-base-java:dev
+
+WORKDIR /airbyte
+
+ENV APPLICATION source-mysql-strict-encrypt
+
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+
+RUN tar xf ${APPLICATION}.tar --strip-components=1
+
+LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.name=airbyte/source-mysql-strict-encrypt
```

Comment:
please, add a new line at the end of file

Reply:
updated it